CMakeLists.txt

```
@@ -996,11 +1021,19 @@ if(BUILD_EXAMPLE)
 	endif()
 endif()
 
+set(PKGCFG_NAME "${PROJECT_NAME}")
+set(PKGCFG_DESC "${PROJECT_NAME} library")
+set(PKGCFG_VERSION "${LIB_VERSION_STRING}")
+set(PKGCFG_LDFLAGS_PUB "-l${BINARY_NAME}")
+
 file(MAKE_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/include/mgba)
 configure_file(${CMAKE_CURRENT_SOURCE_DIR}/src/core/flags.h.in ${CMAKE_CURRENT_BINARY_DIR}/include/mgba/flags.h)
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/res/mgba.pc.in ${CMAKE_CURRENT_BINARY_DIR}/mgba.pc.in @ONLY)
```

Typo I didn't notice earlier:
```suggestion
configure_file(${CMAKE_CURRENT_SOURCE_DIR}/res/mgba.pc.in ${CMAKE_CURRENT_BINARY_DIR}/mgba.pc @ONLY)
```